packages/boba/gateway/src/components/availableBridges/availableBridges.js

```diff
@@ -8,19 +8,20 @@ import { BANXA_URL } from 'util/constant'
 
 function AvailableBridges({ token = null, walletAddress = "" }) {
 
-  const [ bridges, setBridges ] = useState([])
+  const currentNetwork = networkService?.network;
```

Comment:
Good to use selector rather than service.
```suggestion
const currentNetwork = useSelector(selectActiveNetwork());
```